node_collector.sh

```diff
@@ -7,7 +7,7 @@ LOG_PATH=/var/log/cassandra
 
 #List of data directories; if more than one list all with delimiter ','
 #e.g. DATA_PATHS=path/to/dir1,path/to/dir2
-DATA_PATHS=/var/lib/cassandra/data/data
+DATA_PATHS=/var/lib/cassandra/data
```

Comment:
Fix default path

Reply:
Seems default for DSE, apache cassandra, Ubuntu distro all use `/var/lib/cassandra/data` rather than `/var/lib/cassandra/data/data`